radar-commons/src/main/java/org/radarbase/producer/rest/AvroContentConverter.kt

```diff
@@ -25,17 +25,17 @@ class AvroContentConverter(
 
         return coroutineScope {
             val keySchema = async {
-                schemaRetriever.metadata(
+                schemaRetriever.getByVersion(
                     topic = value.topic.name,
                     ofValue = false,
-                    schema = value.topic.keySchema,
+                    version = -1,
                 )
             }
             val valueSchema = async {
-                schemaRetriever.metadata(
+                schemaRetriever.getByVersion(
                     topic = value.topic.name,
                     ofValue = true,
-                    schema = value.topic.valueSchema,
+                    version = -1,
```

Comment:
The POST issue is likely due to a bug here- https://github.com/RADAR-base/radar-commons/blame/4d2044f5a1ec7f7b6b8f4fa1d6433144e7501bac/radar-commons/src/main/java/org/radarbase/producer/schema/SchemaRestClient.kt#L122, this should be calling `schemaGet`, not `schemaPost`.

Reply:
If we update it to a GET request, we should fetch the schemas by version or ID. We don't have either of these at the moment. Should I proceed with -1, which will fetch the latest version?